src/FIOSDK.ts

```diff
@@ -439,6 +440,29 @@ export class FIOSDK {
     return addPublicAddress.execute(this.privateKey, this.publicKey)
   }
 
+  /**
+   * This call allows a any number of public addresses matching the blockchain code, the token code and the public address to be removed from the FIO Address.
+   *
+   * @param fioAddress FIO Address which will be mapped to public address.
+   * @param publicAddresses a list of publicAddresses, each containing chain_code, token_code, and public_address.
```

Comment:
didn't find the `remove_pub_address` in spec (https://stealth.atlassian.net/wiki/spaces/DEV/pages/547487761/API+v1.0), where is documentation for this call?
I assume we don't need to set `public_address` in publicAddresses

Reply:
ill get with pawel and update the spec...thanks..